src/server/models/database.ts

```diff
@@ -0,0 +1,61 @@
+import loki = require("lokijs");
+
+export interface PollInput {
+  creatorName: string;
+  pollName: string;
+  description: string;
+  options: string[];
+}
+export interface Poll extends PollInput {
```

Comment:
Enhancement might be to define `interface PollOption`, and `votes` can be `votes:string[]` instead of `any`, to bring type safety to the rest of the object graph.